tree/30_generic_methods/service_ensure_running_path.cf

```diff
@@ -37,18 +37,10 @@ bundle agent service_ensure_running_path(service_name, service_path)
     "class_prefix"      string => canonify(join("_", "promisers"));
     "args"                          slist => { "${service_name}", "${service_path}" };
 
-  classes:
-    # This method can be called as a substitution if service_ensure_running is called directly, with service_path=service_name
-    # In this case, we don't need to call service_check_running_ps but can use better OS-specific methods if available
-    "real_path"                       not => strcmp("${service_name}", "${service_path}");
```

Comment:
This is an incompatible change, but I see previous like a bug more than a feature...